### DISCUSSION TITLE
Error when trying to make Codespace from PR

### DISCUSSION THREAD

**Discussion by pamelafox** [community/community#57644] [Codespaces]
### Select Topic Area

Bug

### Body

I tried to make a Codespace using the Code dropdown on this PR:
https://github.com/microsoft/sample-app-aoai-chatGPT/pull/34

When I clicked that, it errored:

![Screenshot 2023-06-09 at 10 27 46 AM](https://github.com/community/community/assets/297042/393cbc19-e9fb-47c2-a8bc-f57ac63ca2c2)

So then I went to make it from main using the standard dropdown, and it worked fine.

I then went back to the PR and tried it again, and it seems to work this time.

**Comment by joshspicer** [comment]
This may be a not-so-helpful error message related to the `hostRequirements` added from the PR in the `devcontainer.json`.  That SKU in question has 4gb of memory.

https://github.com/microsoft/sample-app-aoai-chatGPT/pull/34/files#diff-24ad71c8613ddcf6fd23818cb3bb477a1fb6d83af4550b0bad43099813088686R30-R32

**Reply by pamelafox** [reply]
Oh interesting, that does make sense, but I didn't explicitly choose a SKU, so I'm not sure why it auto-picked that.
